source/locale/zh_CN/LC_MESSAGES/reference/generated/optuna.pruners.SuccessiveHalvingPruner.po

```diff
@@ -94,7 +102,8 @@ msgid ""
 " to as :math:`r`). This parameter defaults to 'auto' where the value is "
 "determined based on a heuristic that looks at the number of required "
 "steps for the first trial to complete."
-msgstr ""
+msgstr "一个用于指定分配给 trial 的最小资源的参数（在 `本文 <http://arxiv.org/abs/1810.05934>`_ 中该参数被称为 :math:`r`)."
```

Comment:
A similar paragraph from line 73

Reply:
It's OK as we have line 98 and 108 together is equal to line 73